process/block/metablock.go

```diff
@@ -945,29 +1240,7 @@ func (mp *metaProcessor) createShardInfo(
 		return nil, process.ErrAccountStateDirty
 	}
 
-	if !haveTime() {
```

Comment:
Just think if these prints related to time is up / time elapsed should be moved in createAndProcessCrossMiniBlocksDstMe